bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -127,4 +165,116 @@
 		<description>Battery Power ( + charge, - discharge )</description>
 		<state pattern="%.2f W" readOnly="true"></state>
 	</channel-type>
+
+
+	<channel-type id="meter_model" advanced="true">
+		<item-type>String</item-type>
+		<label>Model</label>
+		<description></description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_serial" advanced="true">
+		<item-type>String</item-type>
+		<label>Serial Number</label>
+		<description>Meter's Serial Number</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_enable" advanced="true">
+		<item-type>Number</item-type>
+		<label>Enabled</label>
+		<description>Enabled</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_location" advanced="true">
+		<item-type>Number</item-type>
+		<label>Location</label>
+		<description>Meter location code</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_currentacphase1">
+		<item-type>Number</item-type>
+		<label>AC Current Phase 1</label>
+		<description></description>
+		<state pattern="%.2f A" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_currentacphase2">
+		<item-type>Number</item-type>
+		<label>AC Current Phase 2</label>
+		<description></description>
+		<state pattern="%.2f A" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_currentacphase3">
+		<item-type>Number</item-type>
+		<label>AC Current Phase 3</label>
+		<description></description>
+		<state pattern="%.2f A" readOnly="true"></state>
+	</channel-type>
```

Comment:
You could re-use one AC current channel type for all phases. The label and description can be set individually in above `channel` tag.

Reply:
Done